lib/rules/array-bracket-spacing.js

```diff
@@ -160,21 +115,37 @@ module.exports = function(context) {
 
 module.exports.schema = [
     {
-        "enum": ["always", "never"]
+        "enum": ["always", "never"],
+        "default": "never"
     },
     {
         "type": "object",
         "properties": {
             "singleValue": {
-                "type": "boolean"
+                "type": "boolean",
+                "default": false
             },
             "objectsInArrays": {
-                "type": "boolean"
+                "type": "boolean",
+                "default": false
             },
             "arraysInArrays": {
-                "type": "boolean"
+                "type": "boolean",
+                "default": false
             }
         },
         "additionalProperties": false
     }
 ];
+
+module.exports.meta = {
+    errors: {
+        extraSpaceAfter: "There should be no space after '{{token}}'",
```

Comment:
I have a hard time believing we'll get people to define their error messages at the very bottom of every file. This is part of the developer ergonomics - it's counterintuitive. Most people define their messages either at the top of the file or right inline with `context.report()`. This represents too dramatic a shift, IMHO. We need to have a better recommendation.